backend/db/schema.rb

```diff
@@ -62,17 +64,6 @@
     t.index ["user_id"], name: "index_participant_metadata_on_user_id"
   end
 
-  create_table "participant_preference", force: :cascade do |t|
-    t.uuid "user_id", null: false
-    t.boolean "cycle_deadlines_email", default: true, null: false
-    t.boolean "prize_cycle_email", default: true, null: false
-    t.boolean "study_available_email", default: true, null: false
-    t.boolean "session_available_email", default: true, null: false
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
-    t.index ["user_id"], name: "index_participant_preference_on_user_id"
-  end
-
   create_table "research_ids", id: :text, force: :cascade do |t|
```

Comment:
looks like our schema.rb was out of date,  this is one of the nice things about a new member bootstrapping the app for the first time, it catches stuff like this :D